src/WinRT.Runtime/Projections.cs

```diff
@@ -77,17 +75,10 @@ static Projections()
             CustomTypeToAbiTypeNameMappings.Add(typeof(System.Type), "Windows.UI.Xaml.Interop.TypeName");
         }
 
+        // Deprecated
         public static void RegisterCustomAbiTypeMapping(Type publicType, Type abiType, string winrtTypeName, bool isRuntimeClass = false)

```

Comment:
Just looking for clarification: Can we not remove this because it's a public / exposed method?

Reply:
Yes, this API is part of the public API surface, so if we were to remove it and someone was calling it, their code would break.  In this case we can potentially do a breaking API compat change given this API was only called in CsWinRT projections built before CsWinRT 1.0 or earlier it seems based on my analysis.  But I am still considering the possibility there might be a future use for this API with some of the generic changes for specialized versions we are considering.